src/main/java/useCases/Timer/TimerInteractor.java

```diff
@@ -27,6 +29,14 @@ public TimerOutputData create(TimerInputData inputData) {
 
         inputDuration = inputData.getInputDurationOfTimer();
         Timer t = timerFactory.create(inputDuration);
+
+        // using the current user service to get the current user object
+        CurrentUserService currentUserService = new CurrentUserService();
```

Comment:
I think the timer interactor should take an instance of the current user service, rather than instantiating a new instance of CurrentUserService

Reply:
Aren't they both in the use case layer? Beacuase if they are, I assume dependency is allowed.